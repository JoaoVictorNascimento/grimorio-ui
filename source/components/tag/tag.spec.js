import '../../../internals/test/helper';

import Tag from './tag-component';

/** @test {Tag} */
describe('Tag component', () => {
/** @test {Tag#render} */
  describe('#render', () => {
    it('render correctly', () => {
      const wrapper = shallow(
        <Tag />
      );
      expect(wrapper.length).toEqual(1);
    });
  });
});
