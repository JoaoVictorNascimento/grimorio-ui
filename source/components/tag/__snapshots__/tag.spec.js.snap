// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag component should render with default props 1`] = `
"<div className=\\"tag primary\\" message=\\"hello\\">
  <Icon className=\\"icon\\" name=\\"cancel\\" size={18} onClick={[undefined]} align=\\"middle\\" />
</div>"
`;
